### ISSUE TITLE
the size is wrong

### ISSUE THREAD

**Issue by Winnie202** [eugenelet/CRFP#1]
Thank you very much for your code contribution, after I run according to your code,like this:
![2022-12-30 16-32-13屏幕截图](https://user-images.githubusercontent.com/86768013/210050515-65c3e8bb-ac66-455e-bf20-b39a2adb89fa.png)

the test data shape is reds, lr image shape is 32*32, hr image shape is 128*128, but an error occurred,like this:

![2022-12-30 16-36-13屏幕截图](https://user-images.githubusercontent.com/86768013/210050855-e5ce8791-96c6-40d7-aafc-d32ef1689ebe.png)
I try to modify some parameters,in model/CRFP.py:

![1](https://user-images.githubusercontent.com/86768013/210051579-36677e53-f219-4a57-8b84-26339145e6be.png)

![2022-12-30 16-42-56屏幕截图](https://user-images.githubusercontent.com/86768013/210051611-18f7eca6-fd3e-4e42-9f7f-cbb1a35a0f90.png)

![3](https://user-images.githubusercontent.com/86768013/210051638-31c95354-b265-4fe7-85c6-1c9612aaf753.png)

but an error occurred,like this:
![2022-12-30 16-46-44屏幕截图](https://user-images.githubusercontent.com/86768013/210051804-0da84cc7-c3f4-4b75-ba96-365987563ee2.png)

Excuse me, where did I go wrong

**Comment by woctezuma** [comment]
Would it make sense to use the following argument?

```
--GT_size 128
```

in order not use the default value of 256:

https://github.com/eugenelet/CRFP/blob/c7d0b82735514ba182c14f188f29fdec390d6a6f/option.py#L88-L89

**Comment by Winnie202** [comment]
> Would it make sense to use the following argument?
> 
> ```
> --GT_size 128
> ```
> 
> in order not use the default value of 256:
> 
> https://github.com/eugenelet/CRFP/blob/c7d0b82735514ba182c14f188f29fdec390d6a6f/option.py#L88-L89

No, after the parameters are modified:--GT_size 128, an error is still reported


![2023-01-02 12-24-11屏幕截图](https://user-images.githubusercontent.com/86768013/210195022-0f65e412-1be0-43f1-9ae4-f747b5c05c25.png)

**Comment by lienfeng1011** [comment]
> > Would it make sense to use the following argument?
> > ```
> > --GT_size 128
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > in order not use the default value of 256:
> > https://github.com/eugenelet/CRFP/blob/c7d0b82735514ba182c14f188f29fdec390d6a6f/option.py#L88-L89
> 
> No, after the parameters are modified:--GT_size 128, an error is still reported
> 
> ![2023-01-02 12-24-11屏幕截图](https://user-images.githubusercontent.com/86768013/210195022-0f65e412-1be0-43f1-9ae4-f747b5c05c25.png)

Hi, could you please check TestSet() class in dataset/reds.py?
Since we source the different preprocessed LR dataset with different scaling factor.
And in test mode we use full size of data and the parameters --GT_size and --LR_size will not be used.
The size could be wrong at the begining when you load those data.
<img width="242" alt="image" src="https://user-images.githubusercontent.com/55732373/211186268-73055857-27e5-4bd0-a6a0-3930909b1490.png">
And I also wonder what are the shapes of these tensors in your case? Thank you.
